dotcom-rendering/src/components/InteractiveBlockComponent.importable.tsx

```diff
@@ -107,6 +107,9 @@ const wrapperStyle = ({
 	background-color: ${themePalette('--interactive-block-background')};
 	min-height: ${getMinHeight(role, loaded)};
 	position: relative;
+	iframe {
+		margin-bottom: -12px;
```

Comment:
@DanielCliftonGuardian @mxdvl I think this is causing an issue in this article https://www.theguardian.com/world/2024/apr/09/us-israel-weapons and others: 

<img width="929" alt="Picture 540" src="https://github.com/guardian/dotcom-rendering/assets/1229808/702cdb27-5370-44f6-a0f1-9d078f8484a9">

Perhaps there's another way to fix this caption issue?

Reply:
Maybe it would have been better to create a flex container, so that we don’t have to content with line-height?